src/main/java/com/unascribed/fabrication/mixin/b_utility/see_items_while_riding/MixinHeldItemRenderer.java

```diff
@@ -0,0 +1,19 @@
+package com.unascribed.fabrication.mixin.b_utility.see_items_while_riding;
+
+import com.unascribed.fabrication.support.EligibleIf;
+import com.unascribed.fabrication.support.Env;
+import com.unascribed.fabrication.support.MixinConfigPlugin;
+import net.minecraft.client.network.ClientPlayerEntity;
+import net.minecraft.client.render.item.HeldItemRenderer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(HeldItemRenderer.class)
+@EligibleIf(anyConfigEnabled={"*.see_items_while_riding", "*.use_items_while_riding"}, envMatches=Env.CLIENT)
+public class MixinHeldItemRenderer {
+	@Redirect(method="updateHeldItems()V", at=@At(value="INVOKE", target="Lnet/minecraft/client/network/ClientPlayerEntity;isRiding()Z"))
```

Comment:
A `Redirect` here makes me a little uneasy. Could this be done with a ModifyVariable?